openfisca_canada/tests/benefitfinder/crb/crb.yaml

```diff
@@ -0,0 +1,49 @@
+- name: "is eligible"
```

Comment:
HI Tom, is it possible to mention what "is eligible" is eligible for so you don't have to look at the file name to see what the tests are about?

Reply:
Good suggestion! I've updated the yaml file